catalyst/dl/callbacks/logging.py

```diff
@@ -288,50 +291,61 @@ def _send_text(self, text: str):
         except Exception as e:
             logging.getLogger(__name__).warning(f"telegram.send.error:{e}")
 
+    def _format_metric(self, name: str, value: float) -> str:
```

Comment:
this function appears 3 times in catalyst. Maybe is it better to move it somewhere?

Reply:
Is it possible to make the `TxtMetricsFormatter._format_metric` static?

Reply:
yes via `@staticmethod`, but it's better to move this func to utils